app/assets/javascripts/Components/criteria_filter.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import {SingleSelectDropDown} from "../DropDownMenu/SingleSelectDropDown";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+
+export class CriteriaFilter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      criterion: "",
```

Comment:
I'd recommend a more descriptive name here, to distinguish between the criterion in the dropdown and the criteria range filters being rendered below it. Maybe `selectedCriterion`